hub/tx_init/rows/acc_sender_gas.tex

```diff
@@ -3,11 +3,11 @@
 	\left\{ \begin{array}{lclr}
 		\accAddress  \high _{i + \locTxInitRoffAccDoingSndPayGas} & = & \locTxInitSenderAddressHi \\
 		\accAddress  \low  _{i + \locTxInitRoffAccDoingSndPayGas} & = & \locTxInitSenderAddressLo \\
-		\multicolumn{3}{l}{
+		\multicolumn{4}{l}{
 		\accTrimAddress
 		{i}{\locTxInitRoffAccDoingSndPayGas}
 		{\locTxInitSenderAddressHi}
-		{\locTxInitSenderAddressLo}} \vspace{2mm} \\
+		{\locTxInitSenderAddressLo}} \vspace{2mm}  \quad (\trash) \\
```

Comment:
These aren't trash, they serve a purpose. We require in the permutation that the very first occurrence of any address trim the address IIRC. For the same reason we also require the trimming during the pre-warming. Not because the address could be malformed, but because we need to make sure that it gets trimmed and because that encounter verifies whether is a precompile or not.